config/templates/metasploit-framework-wrappers/msfupdate.erb

```diff
@@ -100,6 +100,16 @@ EOF
   yum install -y metasploit-framework
 }
 
+install_pkg()
+{
+  (
+    cd ~/Downloads
```

Comment:
Correction: I also had status messages, but I'm leaving them off for now while I nitpick over whitespace.